# For some reason (that I'm working on two different computers?) the my_mods
# often seems to get "grafted" status. I can re-apply my modifications by applying
# this patch file. It was generated by git diff two relevant commits.
# Apply using `patch -p1 -i my_mods.patch` when you're in the root dir.

diff --git a/indent/bib.vim b/indent/bib.vim
index f6c09d3..76ccb60 100644
--- a/indent/bib.vim
+++ b/indent/bib.vim
@@ -57,7 +57,11 @@ function! VimtexIndentBib() abort " {{{1
     if s:count('{', cline) - s:count('}', cline) < 0
       return 0
     else
-      return &sw
+      "return &sw
+      " Avoid indentation for new entries
+      " Not sure if this is a proper fix...
+      "return &sw
+      return ind
     endif
   endif
 
